application/config/packages/security.yaml

```diff
@@ -15,6 +15,20 @@ security:
         login:
             pattern:  ^/user/login$
             security: false
+
+        jwt_login:
+          pattern: ^/api/login_check
```

Comment:
the login check creates a html symfony message if the user does not exist, this should not happen in an API and rather return an appropriate error code:

![grafik](https://github.com/user-attachments/assets/a4b715df-4752-4db4-85c3-102aacbb529f)

Reply:
What should the response look like?  Simple JSON containing an error code?

Reply:
Yes, HTTP Code 401 should remain and IMO as body something like `{"error": "User <username> not found"}` is sufficient.

Reply:
fixed